### ISSUE TITLE
Make Documentation Beginner Friendly

### ISSUE THREAD

**Issue by lazaridis-com** [strapi/strapi#186]
The current Getting Started is at some points misleading. The following would enhance the UX, especially for Beginners:

* Completely separate example for the Strapi Studio, **after** the initial command-line example.
* Remove any insider-knowledge (can be provided in a separate document, like "the technology of strapi"

I you understand (and agree) that documentation must be understandable for beginners (or newcomers from other language domains), then I could provide you with a sample of how the getting started could look.

**Comment by Aurelsicoko** [comment]
@lazaridis-com Thanks, I would love to see what you have in mind, feel free to share! We will rewrite the entire documentation for the next version so I think we have a big challenge here. 

The Strapi Studio will be removed so the getting started will be easiest to understand, IMO. However, we need to put more examples in the new documentation. We also plan to create an example app such as ApolloData did with [GitHunt](https://github.com/apollographql/GitHunt-API).

**Comment by lazaridis-com** [comment]
@Aurelsicoko, you should fix your documentation immediately. I have currently selected LoopBack, but the project has some problems (like excessive amounts of open issues, and that simplicity for newcomers is of no priority, see strongloop/loopback-cli#15) - things that are an indication for deeper problems.

strapi/koa is an alternative to loopback/express. You should make things simpler for beginners, this would give you already a competitive benefit. Additionally, I sense that koa.js has some benefits compared to express.js. Again, you should point thos out, in simple examples, easy to grasp even for beginners.

I'll write a Getting Started Evaluation for strapi (similar for the one written for LoopBack, see lazaridis-com/js-eval#1), this would be the basic sample for your documentation.

(In general, you should remove details, and place them later, in an expert section.)

**Comment by Aurelsicoko** [comment]
Thanks for sharing your thoughts! You're right, we didn't take the time to update the current documentation because we are already focused on the next version. We know that we made mistakes and we're trying to fix them.

Let me know when your JS eval is ready, I will take a look and take in consideration your feedback.

**Comment by lazaridis-com** [comment]
@Aurelsicoko , I have my dev-brain off now, can you please check this:

```
$ strapi new strapitest
$ cd strapitest
$ strapi start

results in:

P:\sand\strapitest>strapi start
info: Server started in P:\sand\strapitest
info: Your server is running at http://localhost:1337
info: Your admin panel is available at http://localhost:1337/admin/
debug: Time: Thu Feb 16 2017 19:54:24 GMT+0200 (GTB Standard Time)
debug: Environment: development
debug: Process PID: 132000
debug: Cluster: master
info: To shut down your server, press <CTRL> + C at any time

error:  TypeError: Promise.defer is not a function
    at Object.exports.create (P:\sand\strapitest\api\user\config\fixtures\route.js:18:28)
error: Bootstrap encountered an error.
error:  TypeError: Promise.defer is not a function
    at Object.exports.create (P:\sand\strapitest\api\user\config\fixtures\route.js:18:28)
(node:132000) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): Error: Callback was already called.
```

**Comment by pierreburgy** [comment]
@lazaridis-com, Strapi v1.x is compatible only with Node 4. But, of course, Strapi v3 will be compatible to the latest Node versions.

**Comment by lazaridis-com** [comment]
> Strapi v1.x is compatible only with Node 4.

makes a negative impression, and at minimum, you should mention that in the docs.

What can I do now? (note: I am emulating a beginner, so don't tell me complicated things)

**Comment by lazaridis-com** [comment]
@pierreburgy , you'll find the mentioned evaluation/documentation here:

lazaridis-com/js-eval#5

**Comment by pierreburgy** [comment]
@lazaridis-com you can use nvm (https://github.com/creationix/nvm) to easily switch between Node versions.

**Comment by lazaridis-com** [comment]
@pierreburgy , you want to point users to a bash-script? What about the windows users? What about the simplicity promise you made? - Any other solution?

(you should possibly first fix the working version/docu, prior to focusing on the new one)

**Comment by Aurelsicoko** [comment]
@lazaridis-com For now, there are no others solutions. We will fix this version issue by releasing a patch. As I already said in my email, could you be more kindly, please? This is open-source, nothing is perfect, we are doing our best to making things better. We are focusing our efforts on the next version, we already told to the community that the current version will not be updated expected if there is a critical issue on it. 

The next version is not an excuse to not patch the current one, we learnt a lot about our mistakes on the v1 and we are trying to fix them. You should also read our blog posts to have a better understanding about the current state of Strapi.

**Comment by lazaridis-com** [comment]
> We will

You should count the word "will" in your sayings/documentation.

> As I already said in my email,

did not get one.

>could you be more kindly, please?

Please, get serious.

Note to readers: aborting evaluation at this point

**Comment by Aurelsicoko** [comment]
@lazaridis-com Your approach is not the good one. You are giving orders. We are here to talk, to share and build something together. We are open to criticals and help. As I said, Strapi is not perfect, and we know that. Instead of just telling that things are bad, the documentation is a bad, you could also offer your help or making more constructive answers to help us.

I'm sorry if we misunderstood each other.

**Comment by lazaridis-com** [comment]
> Your approach is not the good one. You are giving orders.

@Aurelsicoko , Hey, I am in no position to give you orders. It is your very personal problem, if you anticipate my writings as "orders".

> you could also offer your help or making more constructive answers to help us

I have already started to help, by writing a beginners guide. Yet things crash, and you point me to bash-scripts etc. - so I back off, of course.

Anyway, I replied to your email (googles spam-filter has gone a bit nuts lately).

Lets just talk there.
